support-frontend/app/controllers/Application.scala

```diff
@@ -156,6 +156,10 @@ class Application(
       classes = Some(classes)
     )
 
+    val stripeSetupIntentEndpoint =
+      if (stage == Stages.PROD) "https://stripe-intent.support.guardianapis.com/stripe-intent"
+      else "https://stripe-intent-code.support.guardianapis.com/stripe-intent"
```

Comment:
Not related to this PR, but ideally CODE endpoints would be on a separate subdomain or TLD so that there's less chance of a PROD cookie going to a CODE server. Might be worth an audit of our CODE TLDs. I've created a health [task](https://trello.com/c/292FoN2N) on our backlog.

Reply:
We discussed this recently - [see this comment](https://github.com/guardian/support-frontend/pull/2111/files#r331962950)
Ideally it would use `code.support.dev-guardianapis.com`, though this could be updated later?

Reply:
Discussed IRL - we can solve this after it goes live